pkg/diagnostics/report.go

```diff
@@ -40,4 +47,19 @@ func proxyStatus() {
 			fmt.Printf("	❌ %-12s : not set or empty.\n", proxyName)
 		}
 	}
+	fmt.Println("")
+}
+
+// overridedEnvVar retrieves and displays overridden environment variables with the prefix `CONFIG_JS_` from the noobaa-core-0 pod
+func overridenEnvVar(coreApp *appsv1.StatefulSet) {
+	log := util.Logger()
+
+	log.Print("⏳ Retrieving overridden environment variables from the pod `noobaa-core-0`...\n")
```

Comment:
In the first printing, you wrote "Retrieving proxy environment variable details" and here it is "Retrieving overridden environment variables from the pod `noobaa-core-0`", I think it should look the same (either with the part of "from... " or without it - anyway, both of them are retrieved from the StatefulSet.

Reply:
I will update it. Thanks